test/gas/lib/customContractDeploy.js

```diff
@@ -0,0 +1,6 @@
+const deployContract = (abi, accounts, bytecode, web3) => {
```

Comment:
where is this used? why not export it from the base `compileAndDeploy` helper?

Reply:
looks like this duplicates the `deploy` function that's already in `compileAndDeploy`, actually

Reply:
Found where it's used - but this still looks like unnecessary duplication

Reply:
Updated compileAndDeploy to support options.

Reply:
Updated `deploy` to support options.

Reply:
This file was removed.